blockchain/fullblocktests/generate.go

```diff
@@ -1325,7 +1325,7 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.SetTip("bsl5")
 	g.NextBlock("bv15", outs[9], ticketOuts[9], func(b *wire.MsgBlock) {
 		ticketFee := dcrutil.Amount(2)
-		ticketPrice := dcrutil.Amount(g.CalcNextRequiredStakeDifficulty())
+		ticketPrice := dcrutil.Amount(g.CalcNextReqStakeDifficulty(b))
```

Comment:
Needs to be for the block *before* this one so that the difficulty is for this one.